src/Infiltration/ui/SlashGame.tsx

```diff
@@ -25,42 +25,45 @@ const difficulties: {
   Impossible: { window: 150 },
 };
 
-export function SlashGame({ difficulty: _difficulty, onSuccess, onFailure }: IMinigameProps): React.ReactElement {
-  const difficulty: Difficulty = { window: 0 };
-  interpolate(difficulties, _difficulty, difficulty);
+export function SlashGame({ difficulty, onSuccess, onFailure }: IMinigameProps): React.ReactElement {
+  const timingWindow = useMemo(() => {
```

Comment:
here `useMemo` totally makes sense to me 👍

Reply:
Thinking more about it - `useMemo` feels a bit unnecessary here: I checked how `interpolate` works and it's pretty simple.